README.md

```diff
@@ -416,7 +416,7 @@ var options = {
     query: 'alice@example.com'
 };
 
-hkp.lookup(options).then(function(key) {
+hkp.lookup(options).then( async function(key) {
```

Comment:
I do recommend to use either .then or async/await, but not both - it gets messy. Since we do most examples with async/await, the following would be better:
```js
let armoredPubkey = await hkp.lookup(options);
var pubkey = await openpgp.key.readArmored(armoredPubkey);
```

Reply:
Thank you for the feedback, made the changes as recommended